test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/010_Read_only_response_for_set_interior_vehicle_data_if_read_only_params_requested.lua

```diff
@@ -0,0 +1,45 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/3
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/SetInteriorVehicleData.md
+-- Item: Use Case 1: Exceptions: 7.1
+--
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- application sends valid SetInteriorVehicleData with just read-only parameters in "radioControlData" struct for muduleType: RADIO
+-- SDL must
+-- respond with "resultCode: READ_ONLY, success:false" to this application and do not process this RPC.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local module_data_radio = common.getReadOnlyRadioParams()
```

Comment:
@GetmanetsIrina I would propose move function `getReadOnlyRadioParams()` into the script because it is specific (and is used) for the script.